packages/pg-meta/src/sql/functions.ts

```diff
@@ -80,15 +80,16 @@ from
         'mode', t2.mode,
         'name', name,
         'type_id', type_id,
-        'has_default', has_default
+        -- Cast null into false boolean
+        'has_default', COALESCE(has_default, false)
       )) as args
     from
       (
         select
           oid,
           unnest(arg_modes) as mode,
           unnest(arg_names) as name,
-          unnest(arg_types) as type_id,
+          unnest(arg_types)::int8 as type_id,
```

Comment:
Let's keep this as `oid` instead of `int8`

Reply:
Can do but will have to convert the number in zod type to a string, because oid get out as a string with `postgres.`